Data/Sys/GameSettings/r10000000100000002.ini

```diff
@@ -0,0 +1,19 @@
+# 0000000100000002 - Wii Startup Disc Menu
```

Comment:
Sorry I wasn't clear enough. The file should be called 0000000100000002r1.ini (since the format is {$gameid}r{$revision}.ini).